platform/mbed_error.h

```diff
@@ -288,6 +288,7 @@ typedef enum _mbed_module_type {
     MBED_MODULE_DRIVER_PWM,
     MBED_MODULE_DRIVER_QSPI,
     MBED_MODULE_DRIVER_USB,
+    MBED_MODULE_DRIVER_WATCHDOG,
```

Comment:
Seems this line is no longer needed.

Reply:
ok once this goes into feature branch, lets remove it from there +1